Automattic/jetpack
# Update/videopress match feature check criteria

Fixes # 1272-gh-Automattic/greenhouse

#### Changes proposed in this Pull Request:

* Updated the criteria for showing upsells in the Jetpack -> Dashboard for VideoPress to match what is used in Performance -> Media

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Add a Jetpack Premium plan to your test site
* Visit Jetpack -> Dashboard
* You should not see the VideoPress upsell for a paid plan
* Remove the plan and add a plan that doesn't include VideoPress (ex a Security plan with a recent purchase date)
* Visit Jetpack -> Dashboard again, you SHOULD see the upsell
* Try again with no plan, you SHOULD see the upsell
*
